package.json

```
@@ -22,11 +22,12 @@
     "format": "prettier --write \"src/**/*.+(ts|json)\"",
     "test": "jest",
     "test:coverage": "jest --coverage --json --outputFile=jest.results.json && cat ./coverage/lcov.info | coveralls",
+    "test:coverage:html": "jest --silent=false --coverage --coverageReporters html",
     "codegen": "npm run graphql:codegen && npm run swagger:codegen",
     "prepare": "npm run build",
     "prepublishOnly": "npm test && npm run lint",
     "graphql:codegen": "graphql-codegen --config graphql-codegen.yml",
-    "swagger:codegen": " openapi --input https://raw.githubusercontent.com/cowprotocol/services/main/crates/orderbook/openapi.yml --output src/order-book/generated --exportServices false --exportCore false"
+    "swagger:codegen": " openapi --input https://raw.githubusercontent.com/cowprotocol/services/c23f8ac79687622da3d638c9d79e39199eb8ad6a/crates/orderbook/openapi.yml --output src/order-book/generated --exportServices false --exportCore false"
```

Could we match it to a release version (tag) rather than a commit?

```suggestion
    "swagger:codegen": " openapi --input https://raw.githubusercontent.com/cowprotocol/services/v2.117.1/crates/orderbook/openapi.yml --output src/order-book/generated --exportServices false --exportCore false"
```

---

agree, we were discussing here https://github.com/cowprotocol/cow-sdk/pull/136#discussion_r1283323393